docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
 title: API Reference
 ---
 
-## Commands
+## Read Model Connector Interface
 
-### Command HTTP API
+The table below lists functions that a custom Read Model's connector should implement.
 
-A command can be sent using HTTP API.
+| Function Name             | Description                                                                                |
+| ------------------------- | ------------------------------------------------------------------------------------------ |
+| [connect](#connect)       | Initializes a connection to a storage.                                                     |
+| [disconnect](#disconnect) | Closes the storage connection.                                                             |
+| [drop](#drop)             | Removes the Read Model's data from storage.                                                |
+| [dispose](#dispose)       | Forcefully disposes all unmanaged resources that Read Models served by this connector use. |
 
-For instance, to create a new list in the shopping list app:
+### connect
+
+Initializes a connection to a storage. An implementation should return a store object.
+
+#### Arguments
+
+| Argument Name | Description                                       |
+| ------------- | ------------------------------------------------- |
+| readModelName | A read model for which to establish a connection. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#connect)
+```js
+const connect = async readModelName => {
+  fs.writeFileSync(`${prefix}${readModelName}.lock`, true, { flag: 'wx' })
+  readModels.add(readModelName)
+  const store = {
+    get() {
+      return JSON.parse(String(fs.readFileSync(`${prefix}${readModelName}`)))
+    },
+    set(value) {
+      fs.writeFileSync(`${prefix}${readModelName}`, JSON.stringify(value))
+    }
+  }
+  return store
+}
+```
+
+<!-- prettier-ignore-end -->
+
+### disconnect
+
+Closes the storage connection.
+
+#### Arguments
+
+| Argument Name | Description                 |
+| ------------- | --------------------------- |
+| store         | A store object.             |
+| readModelName | A read model to disconnect. |
```

Comment:
"A read model to disconnect" any read model?

Reply:
Changed to 'the'